kyuubi-hive-jdbc/pom.xml

```diff
@@ -168,6 +168,16 @@
             <artifactId>log4j-1.2-api</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.lz4</groupId>
+            <artifactId>lz4-java</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.github.luben</groupId>
+            <artifactId>zstd-jni</artifactId>
```

Comment:
shading lz4 and zstd-jni may cause some problems, have you check if they support relocating native libs like netty does?

Reply:
Refering to zstd doc, (http://facebook.github.io/zstd/)
[aircompressor](https://github.com/airlift/aircompressor) implements zstd compression purely in java. https://github.com/airlift/aircompressor/tree/master/src/main/java/io/airlift/compress/zstd